src/js/game/LevelMVC/LevelView.js

```diff
@@ -1184,6 +1198,22 @@ module.exports = class LevelView {
   }
 
   refreshActionPlane(positions) {
+    // We need to add indices to refresh if there are other blocks in the action plane that might
+    // conflict with the drawing of refreshed blocks.
+    for (let i = 0; i < positions.length; ++i) {
+      const positionBelow = [positions[i][0], positions[i][1] + 1];
+      const indexIsValid = this.controller.levelModel.actionPlane.inBounds(positionBelow);
+      if (indexIsValid) {
+        const positionOfInterest = [positions[i][0], positions[i][1] + 1];
```

Comment:
nit: why not reuse `positionBelow`?

Reply:
Becaaause there was another step and then I re-wrote some of it, and didn't realize that the logic became duplicated 😦